eventbridge-pipes-amazonmq-to-eventbridge-sam/eventbridge-pipes-amazonmq-to-eventbridge-sam-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "Amazon MQ broker as a producer for a Amazon EventBridge Message Bus using Amazon EventBridge Pipes",
```

Comment:
Title exceeds the required 75 character limit

Reply:
Adjusted length to 74 characters in commit 9175bc0dbb7bf99a2290fc4e9246968277135cc1